store/sqlstore/preference_store.go

```diff
@@ -140,6 +140,23 @@ func (s SqlPreferenceStore) Get(userId string, category string, name string) (*m
 	return &preference, nil
 }
 
+func (s SqlPreferenceStore) GetCategoryAndName(category string, name string) (model.Preferences, error) {
```

Comment:
This should have been `GetByCategoryAndName` as that's how we do it elsewhere, but I do see that we have a `GetCategory` already. So good for now.